lib/actions-oclif/os/configure.ts

```diff
@@ -144,6 +150,11 @@ export default class OsConfigureCmd extends Command {
 		version: flags.string({
 			description: 'balenaOS version, for example "2.32.0" or "2.44.0+rev1"',
 		}),
+		'system-connection': flags.string({
+			multiple: true,
+			required: false,
+			description: 'paths to local files to place into the \'system-connections\' directory',
```

Comment:
* Let's add something like `char: 'c'` here (equivalent to Capitano's 'alias') so one can write:
```
... -c a.con -c b.con -c c.con
```
instead of
```
... --system-connection a.con --system-connection b.con --system-connection c.con
```

* It would be nice to add a few words to the output of `balena help os configure`, explaining that additional NetworkManager connection profiles can be added using the `--system-connection` option. It could also mention the filename that gets generated by default for WiFi settings taken from `--config-wifi-key` and `--config-wifi-ssid`. (What happens if the user provides that same filename as a `--system-connection` argument? Looks like `--system-connection` overwrites the 'default' filename, which is OK.)

Reply:
so the `os configure` command puts the wifi settings into the `config.json` and doesn't create the `system-connection` file; this would be populated to a file out on boot.

I thought that some help was already added? If you're referring to the main bulk of the text, then sure 👍

The filenames are copied over, so yes, it would overwrite anything that existed. I think this is OK since it's a pre-configuration step.

Reply:
The main bulk of the text, yes. It seems like a big feature that should be explained; this is (partially?) addressing 3 CLI issues often requested by users and support agents:
* https://github.com/balena-io/balena-cli/issues/957
* https://github.com/balena-io/balena-cli/issues/1162
* https://github.com/balena-io/balena-cli/issues/1498
* Others?

So it would be good to explain that if users want to configure additional interfaces to the default WiFi of interactive prompts, they can use the `--system-connection` option to point to NetworkManager profiles that the user wrote. And point to documentation that describes the file format and such, like https://www.balena.io/docs/reference/OS/network/2.x/#wifi-setup